lib/scss_lint/linter/README.md

```diff
@@ -1457,6 +1465,8 @@ Reports lines containing trailing whitespace.
 
 ## TrailingZero
 
+**Disabled by default**
```

Comment:
I'm actually surprised this is disabled by default. Unfortunately I didn't think to explain why I disabled it by default in the original commit 800a67bb.